features/contrast-color.yml

```diff
@@ -0,0 +1,8 @@
+name: contrast-color()
+description: "The `contrast-color()` CSS function picks a color that has guaranteed contrast with the color passed as an argument."
+spec: https://drafts.csswg.org/css-color-5/#contrast-color
+group: color-types
+compat_features:
+  # BCD is out of date with the spec.
+  # See https://github.com/mdn/content/issues/36904
+  - css.types.color.color-contrast
```

Comment:
As I look at this more, it seems like both the name and the functionality are shifting (and the level 6 specification is very different). What if we mark `css.types.color.color-contrast` as either non-standard_track or deprecated, and then treat this feature similarly to #2301, as a placeholder for future expected functionality?

Reply:
I opened up https://github.com/mdn/browser-compat-data/pull/25305 to specify `contrast-color` as non-standard track. This solves the immediate problem of how to represent it as a feature, but doesn't address the question of reserving keys for future functionality.

Reply:
@captainbrosset FYI, `css.types.color.color-contrast` has been removed from BCD.